src/webui/abstractwebapplication.cpp

```diff
@@ -110,7 +110,7 @@ Http::Response AbstractWebApplication::processRequest(const Http::Request &reque
     header(Http::HEADER_X_FRAME_OPTIONS, "SAMEORIGIN");
     header(Http::HEADER_X_XSS_PROTECTION, "1; mode=block");
     header(Http::HEADER_X_CONTENT_TYPE_OPTIONS, "nosniff");
-    header(Http::HEADER_CONTENT_SECURITY_POLICY, "default-src 'self' 'unsafe-inline' 'unsafe-eval';");
+    header(Http::HEADER_CONTENT_SECURITY_POLICY, "default-src 'self'; script-src 'self' 'unsafe-inline'; img-src 'self' data:; style-src 'self' 'unsafe-inline'; object-src 'none';");
```

Comment:
OK, this is much better, just one minor thing, can you please place the directives in the following order:
```
default-src; style-src; img-src; script-src; object-src;
```